wca-regulations.md

```diff
@@ -516,7 +516,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If such stands are used, the organization team should make an announcement before the round starts.
         - F2a1) The judge must remove the stand from the mat immediately after the competitor removes the puzzle from it.
 - F3) At the end of the inspection period, the competitor places the puzzle in a standing position. Penalty: disqualification of the attempt (DNF).
-    - F3a) The competitor must not change the positions of any pins from their scrambled positions before the start of the solve. Penalty: disqualification of the attempt (DNF).
+    - F3a) The competitor must not change the positions of any pins from their scrambled positions before the start of the solve. Penalty: time penalty (+2 seconds).
```

Comment:
Nit: I'd spell this out here:
```suggestion
    - F3a) The competitor must not change the positions of any pins from their scrambled positions before the start of the solve. Penalty for any number of pin changes: time penalty (+2 seconds).
```

Reply:
We won't be implementing this change for this cycle, so we won't have to worry about this PR